tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -33,110 +33,195 @@
 import config
 import mle
 import thread_cert
+from pktverify.consts import WIRESHARK_OVERRIDE_PREFS, ADDR_ERR_URI, ADDR_QRY_URI, ADDR_NTF_URI, MLE_CHILD_ID_REQUEST, MLE_CHILD_ID_RESPONSE, REALM_LOCAL_ALL_ROUTERS_ADDRESS, NL_ML_EID_TLV, NL_TARGET_EID_TLV, NL_RLOC16_TLV, COAP_CODE_POST
+from pktverify.packet_verifier import PacketVerifier
+from pktverify.addrs import Ipv6Addr
 
 DUT_LEADER = 1
 ROUTER1 = 2
 ROUTER2 = 3
 MED1 = 4
 SED1 = 5
-MED3 = 6
+MED2 = 6
 
-MTDS = [MED1, SED1, MED3]
+MTDS = [MED1, SED1, MED2]
+ON_MESH_PREFIX = '2001::/64'
+IPV6_ADDR = '2001::1234'
+
+# Test Purpose and Description:
+# -----------------------------
+# The purpose of this test case is to validate the DUT’s ability to perform
+# duplicate address detection.
+#
+# Test Topology:
+# -------------
+# MED_2 - Leader(DUT)
+#         /     \
+#  Router_1   Router_2
+#      |          |
+#    MED_1       SED
+#
+# DUT Types:
+# ----------
+#  Leader
 
 
 class Cert_5_3_7_DuplicateAddress(thread_cert.TestCase):
+    USE_MESSAGE_FACTORY = False
+
     TOPOLOGY = {
         DUT_LEADER: {
+            'name': 'LEADER',
             'mode': 'rdn',
             'panid': 0xface,
-            'allowlist': [ROUTER1, ROUTER2, MED3]
+            'allowlist': [ROUTER1, ROUTER2, MED2]
         },
         ROUTER1: {
+            'name': 'ROUTER_1',
             'mode': 'rdn',
             'panid': 0xface,
             'router_selection_jitter': 1,
             'allowlist': [DUT_LEADER, MED1]
         },
         ROUTER2: {
+            'name': 'ROUTER_2',
             'mode': 'rdn',
             'panid': 0xface,
             'router_selection_jitter': 1,
             'allowlist': [DUT_LEADER, SED1]
         },
         MED1: {
+            'name': 'MED_1',
             'is_mtd': True,
             'mode': 'rn',
             'panid': 0xface,
             'allowlist': [ROUTER1]
         },
         SED1: {
+            'name': 'SED',
             'is_mtd': True,
             'mode': '-',
             'panid': 0xface,
             'allowlist': [ROUTER2]
         },
-        MED3: {
+        MED2: {
+            'name': 'MED_2',
             'is_mtd': True,
             'mode': 'rn',
             'panid': 0xface,
             'allowlist': [DUT_LEADER]
         },
     }
 
+    # override wireshark preferences with case needed parameters
+    CASE_WIRESHARK_PREFS = WIRESHARK_OVERRIDE_PREFS
```

Comment:
where is the code that uses `CASE_WIRESHARK_PREFS` ?

BTW, should use `copy.deepcopy` to avoid affecting other test cases?